.github/workflows/conformance.yml

```diff
@@ -22,7 +22,8 @@ on:
   pull_request:
     branches:
       - master
-      - release-*
+      - 'release-*'
+      - 'feature/*'
```

Comment:
@dapr/maintainers-components-contrib Please let me know if this enable is OK ... This will only run PR conformance tests for PRs targeting feature branch also ...

Reply:
I suppose this is fine as it will only run the Docker bases conformance tests. Everything else requires manual `/ok-to-test`. Not all Azure tests perform well with multiple runs in parallel.